### PR TITLE
New: Added Firefox prefs and args to enable customization

### PR DESCRIPTION
**Pull request by mmusil**

### CODE REVIEW

File: src/webdriver_kaifuku/__init__.py [hunk 1]
```diff
@@ -100,8 +101,23 @@ def _config_options_for_remote_firefox(browser_conf: dict) -> webdriver.FirefoxO
         desired_capabilities = browser_conf.get("webdriver_options", {}).get(
             "desired_capabilities", {}
         )
+        desired_capabilities_firefox_options = desired_capabilities.pop("firefoxOptions", {})
```
Comment (RonnyPfannschmidt) [inline src/webdriver_kaifuku/__init__.py]:
```suggestion
        desired_capabilities_firefox_options = desired_capabilities.get("firefoxOptions", {})
```

**Comment by mmusil** [comment]
@RonnyPfannschmidt I know that you are on paternity leave and Dmitrii has left, but do you know if we could move forward with this PR in any way?

**Comment by RonnyPfannschmidt** [comment]
@mmusil thanks for the ping, i had this in the wrong todo bin

I'll be back in office in next Monday
With some luck I may be able to get back to this earlier

**Comment by mmusil** [comment]
Awesome, thanks! I wasn't sure if you will see the ping at all... 😅 It would be good to catch up on things once you get back. :)

**Pull request merged**